### PR TITLE
LPS-100534 | Apply new Session utilites in portal-web

### PR DESCRIPTION
**Pull request by mateomustapic**

### CODE REVIEW

**Comment by jbalsas** [comment]
Hey @mateomustapic, the diffs in this PR look way off... how come we're changing so many lines?

I guess you're somehow formatting them automatically, but this shouldn't happen. Could you try to figure out what's wrong and resend?

Thanks!

File: portal-web/docroot/html/common/themes/bottom_js_logging.jspf [hunk 1]
```diff
@@ -1,69 +1,69 @@
...
+String collapsed = HtmlUtil.escapeJS(GetterUtil.getString(SessionClicks.get(request, "liferay_js_console_collapsed", null), "false"));
+%>
+
+<div id="liferayJSConsole"></div>
+
+<aui:script>
+	AUI().use(
+		'console',
+		'console-filters',
+		'liferay-store',
```
Comment (jbalsas) [inline portal-web/docroot/html/common/themes/bottom_js_logging.jspf]: Is `liferay-store` still needed after the removal?

File: portal-web/docroot/html/taglib/ui/panel_container/javascript.jspf [hunk 2]
```diff
@@ -1,67 +1,69 @@
...
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<aui:script use="aui-toggler,liferay-store">
```
Comment (jbalsas) [inline portal-web/docroot/html/taglib/ui/panel_container/javascript.jspf]: I guess `liferay-store` is no longer needed?

File: portal-web/docroot/html/taglib/ui/toggle/page.jsp [hunk 3]
```diff
@@ -1,108 +1,107 @@
...
+	</c:when>
+	<c:otherwise>
+		<img alt="<liferay-ui:message escapeAttribute="<%= true %>" key="toggle" />" id="<%= id %>_image" onclick="<%= stateVar %>Toggle();" src="<%= defaultImage %>" style="margin: 0px;" />
+	</c:otherwise>
+</c:choose>
+
+<aui:script>
+	var <%= stateVar %> = '<%= defaultStateValue %>';
+
+	Liferay.provide(
```
Comment (jbalsas) [inline portal-web/docroot/html/taglib/ui/toggle/page.jsp]: If we no longer need a dependency, we don't need to wrap the method in a `Liferay.provide`

**Comment by mateomustapic** [comment]
> Hey @mateomustapic, the diffs in this PR look way off... how come we're changing so many lines?
> 
> I guess you're somehow formatting them automatically, but this shouldn't happen. Could you try to figure out what's wrong and resend?
> 
> Thanks!

I will check now and resend fixed PR

**Pull request closed without merging**